bundle/pom.xml

```diff
@@ -88,10 +87,6 @@
                                 <pattern>io.netty</pattern>
                                 <shadedPattern>software.amazon.awssdk.thirdparty.io.netty</shadedPattern>
                             </relocation>
-                            <relocation>
-                                <pattern>com.typesafe</pattern>
-                                <shadedPattern>software.amazon.awssdk.thirdparty.com.typesafe</shadedPattern>
-                            </relocation>
```

Comment:
We still need this, right?

Reply:
No we shouldn't need it, the source imported version is already at an internal package https://github.com/aws/aws-sdk-java-v2/blob/2dda00780700fbe3c616d0ec5e733d83aafd3ba5/http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/nrs/package-info.java#L16-L24